### PR TITLE
firefly-perfnode and ffperf Chart

### PR DESCRIPTION
**Pull request by hfuss**
Used for install FireFly nodes onto K8s for performance testing and then running distributed instances of `ffperf` against them

Related to #26

### CODE REVIEW

File: charts/firefly-perfnode/values.yaml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+              storage: 10Gi
+      - metadata:
+          name: postgres
+        spec:
+          accessModes:
+            - ReadWriteOnce
+          storageClassName: io2
+          resources:
+            requests:
+              storage: 10Gi
```
Comment (drewmarshburn) [inline charts/firefly-perfnode/values.yaml:143]: Sounds like based on preliminary testing, 10GB may not be a good default.

File: charts/firefly-perfnode/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v2
+name: firefly-perfnode
+description: |
+  A Helm chart for deploying FireFly nodes configured for performance testing flows between members.
+type: application
+version: 0.1.0
+appVersion: "0.14.0"
```
Comment (calbritt) [inline charts/firefly-perfnode/Chart.yaml:23]: Should this be 14.1 now?

File: charts/firefly-perfnode/values.yaml [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+    service:
+      httpPort: 3000
+      adminPort: 3001
+      debugPort: 3030
+      metricsPort: 3100
+
+    initContainers: ""
+
+    # Because the swarm key is hardcoded, multiple firefly _networks_ installed on the same k8s cluster via
```
Comment (drewmarshburn) [inline charts/firefly-perfnode/values.yaml:49]: Are the perf tests currently running with multiple FF networks on a single cluster?

**Review by drewmarshburn — APPROVED** [review]
Looks good thus far, one small question.

File: charts/firefly-perfnode/values.yaml (hunk 3 above)
Comment (hfuss) [reply charts/firefly-perfnode/values.yaml:49]: Currently no, and bc multicast doesn't work on K8s by default this is not much of an issue. IPFS nodes have to be told at least one other member's IP in order to cluster, so two clusters accidentally joining each other isn't really possible w/o manual intervention.

**Comment by nguyer** [comment]
@hfuss Should the `charts/firefly-perfnode/charts/firefly-0.4.1-20220324-129.tgz` file be checked into git?

**Review by nguyer — APPROVED** [review]

**Pull request merged**